### ISSUE TITLE
Servicenow integration

### ISSUE THREAD

**Issue by rguptarg** [apache/skywalking#3745]
Hi Team,

We are planning to implement skywalking in our infra for APM. so we want skywalking integration with Service now for ticketing.

So please let me, is it possible at skywalking end. can we forwards SNMP traps to any other tools ?

**Comment by wu-sheng** [comment]
What do you mean `SNMP`? And what is the meaning of `Service now for ticketing`? Hard to follow

**Comment by rguptarg** [comment]
We are using Servicenow as a ticketing tools, where we forwards SNMP traps from any source and Servicenow will raise a ticket and assign to respective team.

So is it possible in skywalking ?

If there is any Mail notification features, it will also helpful

**Comment by wu-sheng** [comment]
Notification for what? That is my question.

**Comment by rguptarg** [comment]
Alerting Notification, If I have configured any alert condition in skywalking, how can I notify to respective team?

I am new in skywalking, so not fully aware about skywalking functionality.

**Comment by wu-sheng** [comment]
We have webhook already for alerting. So, you could build a layer to forward and transform the format again.
